includes/admin/class-wc-rest-payments-settings-controller.php

```diff
@@ -556,6 +558,23 @@ private function update_is_multi_currency_enabled( WP_REST_Request $request ) {
 		update_option( '_wcpay_feature_customer_multi_currency', $is_multi_currency_enabled ? '1' : '0' );
 	}
 
+	/**
+	 * Updates customer Multi-Currency feature status.
```

Comment:
Looks like the method description needs to be updated. What do you think about this for the description:

> Updates the '_wcpay_feature_client_secret_encryption' option.